plugins/snowflake/dbt/include/snowflake/macros/adapters.sql

```diff
@@ -156,3 +172,19 @@
     alter table {{ relation }} alter {{ adapter.quote(column_name) }} set data type {{ new_column_type }};
   {% endcall %}
 {% endmacro %}
+
+{% macro snowflake__alter_relation_comment(relation, relation_comment) -%}
+  comment on table {{ relation }} IS $${{ relation_comment | replace('$', '[$]') }}$$;
```

Comment:
Can we use `relation.type` instead of `table` here? That will let us use this macro for both tables and views :)

Same for the implementation of `snowflake__alter_column_comment` below!